packages/blocks/src/page-block/edgeless/components/toolbar/edgeless-toolbar.ts

```diff
@@ -98,10 +102,47 @@ export class EdgelessToolbar extends WithDisposable(LitElement) {
     .transform-button:hover svg {
       transform: scale(1.15);
     }
+
+    .edgeless-present-frame {
+      width: 145px;
+      text-align: center;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+
+    .edgeless-prensent-frame-title {
```

Comment:
Should be `.edgeless-frame-navigator-title`. Please remove all the *present* vocabularies here (and replace them as *frame navigator*), since *present* it's a verb that doesn't fit well, and its noun form means gift, which is also wrong in our case.